uk.co.compucorp.civicrm.hrleaveandabsences/tests/phpunit/CRM/HRLeaveAndAbsences/BAO/AbsenceTypeTest.php

```diff
@@ -39,6 +39,14 @@ public function testTypeTitlesShouldBeUnique() {
     AbsenceTypeFabricator::fabricate(['title' => 'Type 1']);
   }
 
+  public function testTypeTitleShouldNotBeEmpty() {
```

Comment:
There is no need for type check.

Reply:
@ajesamson please ensure **all** cases listed in `$params` are executed. It may stop the test once the first exception is thrown. I am not sure, but let's check this.